src/test/java/domain/PositionTest.java

```diff
@@ -0,0 +1,89 @@
+package domain;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class PositionTest {
+
+    @Nested
+    @DisplayName("성공 케이스")
+    class SuccessCases {
+
+        @Test
+        @DisplayName("값이 같은 Position은 동등하다")
+        void equalWhenSameValue() {
+            Position p1 = new Position(2);
+            Position p2 = new Position(2);
+
+            Assertions.assertThat(p1).isEqualTo(p2);
+            Assertions.assertThat(p1).isNotSameAs(p2);
+            Assertions.assertThat(p1.getValue()).isEqualTo(2);
+        }
+
+        @Test
+        @DisplayName("값이 같은 domain.Position 객체는 양방향 equals 비교 시 true를 반환한다")
+        void positionEqualityIsSymmetric() {
+            Position a = new Position(5);
+            Position b = new Position(5);
+
+            Assertions.assertThat(a).isEqualTo(b);
+            Assertions.assertThat(b).isEqualTo(a);
+        }
+
+        @Test
+        @DisplayName("동등한 Position은 hashCode도 동일하다")
+        void hashCodeShouldMatchIfEqual() {
+            Position a = new Position(4);
+            Position b = new Position(4);
+
+            Assertions.assertThat(a).isEqualTo(b);
+            Assertions.assertThat(a.hashCode()).isEqualTo(b.hashCode());
+        }
+
+        @Test
+        @DisplayName("forward는 기존 값보다 1 증가한 새로운 Position을 반환한다")
+        void forwardCreatesNewPosition() {
+            Position origin = new Position(0);
+            Position moved = origin.forward();
+
+            Assertions.assertThat(moved).isEqualTo(new Position(1));
+            Assertions.assertThat(moved).isNotSameAs(origin);
+            Assertions.assertThat(origin.getValue()).isEqualTo(0);
+            Assertions.assertThat(moved.getValue()).isEqualTo(1);
```

Comment:
`Position origin = new Position(0);` 에서 이미 첫 시작이 0인 것을 알고 있으니까, moved의 position이 1임만을 검증하면 충분하지 않을까요?
```suggestion
            Position origin = new Position(0);
            Position moved = origin.forward();

            Assertions.assertThat(moved.getValue()).isEqualTo(1);
```

Reply:
피드백 감사합니다~! 🙇‍♂️  

말씀해주신 것처럼 `Position origin = new Position(0)`이 명확한 초기 상태이기 때문에,  
`forward()` 이후 `1`이 되었는지만 확인하면 핵심 로직 검증은 충분하다는 점에 공감이 됩니다!

처음엔 불변 객체로서의 특성과 내부 상태 변화 유무도 함께 확인하고 싶다는 생각에  
`isNotSameAs`, `origin.getValue()` 등 추가적인 검증을 넣었었는데,  
지금 보니 **과도한 검증으로 테스트가 오히려 복잡해진 것 같아요.**

핵심 규칙만 명확히 드러내는 방향으로 테스트를 간결하게 정리해보겠습니다!  
꼼꼼한 피드백 덕분에 테스트의 목적을 더 명확히 바라볼 수 있었어요. 감사합니다 😊